test/fake_device_factory_test.rb

```diff
@@ -21,11 +21,62 @@
 # find current contact information at www.suse.com.
 
 require_relative "spec_helper"
-require "storage"
-require "y2storage/fake_device_factory"
 
 describe Y2Storage::FakeDeviceFactory do
 
+  it "reads yaml of simple dasd and partition setup" do
+
+    environment = Storage::Environment.new(true, Storage::ProbeMode_NONE, Storage::TargetMode_DIRECT)
+    storage = Storage::Storage.new(environment)
+    staging = storage.staging
+
+    # rubocop:disable Style/StringLiterals
+
+    input = ['---',
+             '- dasd:',
+             '    name: "/dev/sda"',
+             '    size: 256 GiB',
+             '    block_size: 4 KiB',
+             '    partition_table: dasd',
```

Comment:
is this line really needed? It should not be. A `disk` can be created without that (which defaults to a MSDOS partition table, if I'm not wrong). Creating a `dasd` without that line makes even more sense, since there is only a valid type.